apparmor.d/abstractions/bus/org.freedesktop.UPower

```diff
@@ -15,10 +15,11 @@
   dbus send bus=system path=/org/freedesktop/UPower/devices/DisplayDevice
        interface=org.freedesktop.DBus.Properties
        member={Get,GetAll}
-       peer=(name=org.freedesktop.UPower, label=upowerd),
-  dbus send bus=system path=/org/freedesktop/UPower/devices/DisplayDevice
+       peer=(name=:*, label=upowerd),
+
+  dbus receive bus=system path=/org/freedesktop/UPower/devices/{mouse_hidpp_battery_0,battery_BAT0}
```

Comment:
`/org/freedesktop/UPower/devices/*` otherwise it depends of the device type too much.